wmde/FundraisingFrontend
# Fix subscription process

Bug: T177937

Add missing templates for subscription success use case. Fix the broken confirmation link sent in the confirmation mail.

To test locally:
- Go to http://localhost:8082/contact/subscribe?skin=10h16 and type whatever email address in the form.
- Go to http://localhost:8025/ (mail hog) and open the last mail.
- Copy the confirmation link, paste it in the url bar and add `?skin=cat17` at the end.  

You should see the successful confirmation page `Subscription_Confirmed.html.twig`. If you go back and click on the link from the email, you should see the error page `Warning_Page.html.twig`, because it was already confirmed.
